app/main.py

```diff
@@ -2,4 +2,61 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: str = None) -> None:
```

Comment:
Invalid type hint for `coords`. `coords` is a list
```suggestion
    def __init__(self, name: str, weight: int, coords: list[int] = None) -> None:
```